src/Sandbox/World/Chunks/Structure/Chunk.cpp

```diff
@@ -0,0 +1,30 @@
+#include "Chunk.h"
+
+
+Chunk::Chunk(const ChunkFrame& frame, ChunkBlocks blocks, Camera& camera)
+	: _frame(frame), _blocks(std::move(blocks)), _camera(camera)
+{
+}
+
+void Chunk::Draw() const
+{
+	for (auto& block : _blocks.block)
+	{
+		auto origin = block.first;
+
+		const auto& faceModel = block.second.model;
```

Comment:
This is not face model this is block model.

Reply:
done.